docs/reference/remap/is_nullish.cue

```diff
@@ -0,0 +1,39 @@
+package metadata
+
+remap: functions: is_nullish: {
+	arguments: [
+		{
+			name:        "value"
+			description: #"The value to check for "nullishness", i.e. a useless value."#
+			required:    true
+			type: ["string", "null"]
```

Comment:
Noting that we expanded this to accept any type.

Reply:
Good call. Updated.